plugin/wavesurfer.regions.js

```diff
@@ -331,6 +331,9 @@ WaveSurfer.Region = {
             my.element.addEventListener('mousedown', onDown);
             my.wrapper.addEventListener('mouseup', onUp);
             my.wrapper.addEventListener('mousemove', onMove);
+            document.querySelector("body").addEventListener("mouseup", function () {
```

Comment:
In function `onUp` you do `e.stopPropagation();` and `e.preventDefault();`. This applied to the body can make some functionality of the web using this library to brake, that's why I put it this way. But If we remove `my.wrapper.addEventListener('mouseup', onUp);` we can delete the two lines that stop event bubbling.